### DISCUSSION TITLE
slow install

### DISCUSSION THREAD

**Discussion by sabhattrai** [Cyfrin/foundry-full-course-cu#1661] [General]
take so much time while initilizing the foundry project using " forge init --force".and give  a error
```
warning: LF will be replaced by CRLF in .vscode/extensions/ms-python.vscode-pylance-2023.10.53/dist/typeshed-fallback/stubs/redis/redis/asyncio/parser.pyi.
```

**Comment by EngrPips** [comment]
Hello @sabhattrai, That seems more like a warning and not an error and if installation takes longer that is on your internet connection. You can maybe retry when your internet connection is stable.

**Reply by sabhattrai** [reply]
@EngrPips  but other website and other dependencies of react are easily installed. only got this problem while initilizing the foundry project

**Reply by sabhattrai** [reply]
@PatrickAlphaC  need a help for avobe problem

**Reply by EngrPips** [reply]
I see. I didn't run into any error when I tried installing Foundry, making it hard to know what could be wrong.
